insalan/tournament/management/commands/populate.py

```diff
@@ -144,6 +146,15 @@ def generate_tournaments(self, events, games):
                     game=Game.objects.get(id=game_id),
                     name="tourney_" + generate_garbage(randint(18, 26)),
```

Comment:
btw this line breaks database constraints as well (it's the only broken one at the moment from what i've seen)

Reply:
Fixed in a0f468e4f8a426337f12a21f98584bcf8a1fec53